<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AVSettingsDialog</class>
 <widget class="QDialog" name="AVSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio/ Video Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="4">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Aspect</string>
       </property>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="QComboBox" name="SignalSelectComboBox"/>
     </item>
     <item row="1" column="6">
      <widget class="QComboBox" name="DualMonoComboBox"/>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Adv. Video Adjust</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QComboBox" name="AdvancedVideoAdjustComboBox"/>
     </item>
     <item row="13" column="4">
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>Sound Delay (frames)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QLabel" name="label_32">
       <property name="text">
        <string>Dual Mono</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="AspectComboBox"/>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="BlackSetupComboBox"/>
     </item>
     <item row="15" column="4">
      <widget class="QLabel" name="label_30">
       <property name="text">
        <string>Dialog Enhancement</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Brightness</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>BNR</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>MNR</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="label_46">
       <property name="text">
        <string>Sharpness</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="label_45">
       <property name="text">
        <string>Detail</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Hue</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>YNR</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="ResolutionComboBox"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Resolution</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Chroma Level</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Contrast</string>
       </property>
      </widget>
     </item>
     <item row="15" column="3">
      <widget class="QComboBox" name="DialogEnhancementComboBox"/>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>CNR</string>
       </property>
      </widget>
     </item>
     <item row="9" column="4">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Signal Select</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Black Setup</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>MCACC Memory Set</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Pure Cinema</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QComboBox" name="PhaseControlComboBox"/>
     </item>
     <item row="4" column="0">
      <widget class="QComboBox" name="StreamSmootherComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="PureCinemaComboBox"/>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Prog. Motion</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Stream Smoother</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QComboBox" name="MemSetNoComboBox"/>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Phase Control</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QSlider" name="DetailSlider">
         <property name="maximum">
          <number>8</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="DetailLabel">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="9" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QSlider" name="MNRSlider">
         <property name="maximum">
          <number>8</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="MNRLabel">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSlider" name="CNRSlider">
         <property name="maximum">
          <number>8</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="CNRLabel">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="13" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QSlider" name="HueSlider">
         <property name="minimum">
          <number>-6</number>
         </property>
         <property name="maximum">
          <number>6</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="HueLabel">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="5" rowspan="17">
      <widget class="QFrame" name="frame_2">
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::VLine</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="17">
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::VLine</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item row="13" column="3">
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QSlider" name="SoundDelaySlider">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="SoundDelayLabel">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="12" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QSlider" name="BrightnessSlider">
         <property name="minimum">
          <number>-6</number>
         </property>
         <property name="maximum">
          <number>6</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="BrightnessLabel">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="14" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QSlider" name="ContrastSlider">
         <property name="minimum">
          <number>-6</number>
         </property>
         <property name="maximum">
          <number>6</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ContrastLabel">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QSlider" name="BNRSlider">
         <property name="maximum">
          <number>8</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="BNRLabel">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSlider" name="YNRSlider">
         <property name="maximum">
          <number>8</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="YNRLabel">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSlider" name="ProgMotionSlider">
         <property name="minimum">
          <number>-4</number>
         </property>
         <property name="maximum">
          <number>4</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ProgMotionLabel">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="11" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QSlider" name="SharpnessSlider">
         <property name="maximum">
          <number>8</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="SharpnessLabel">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="15" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QSlider" name="ChromaLevelSlider">
         <property name="minimum">
          <number>-6</number>
         </property>
         <property name="maximum">
          <number>6</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ChromaLevelLabel">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="6">
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QSlider" name="EffectStrengthSlider">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="EffectStrengthLabel">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="9" column="6">
      <widget class="QComboBox" name="HDMIOutputComboBox"/>
     </item>
     <item row="10" column="6">
      <widget class="QComboBox" name="HDMIAudioComboBox"/>
     </item>
     <item row="12" column="6">
      <widget class="QComboBox" name="DimmerComboBox"/>
     </item>
     <item row="13" column="6">
      <widget class="QComboBox" name="SleepComboBox"/>
     </item>
     <item row="7" column="7">
      <widget class="QLabel" name="label_36">
       <property name="text">
        <string>Effect Strenth</string>
       </property>
      </widget>
     </item>
     <item row="9" column="7">
      <widget class="QLabel" name="label_40">
       <property name="text">
        <string>HDMI Output</string>
       </property>
      </widget>
     </item>
     <item row="10" column="7">
      <widget class="QLabel" name="label_41">
       <property name="text">
        <string>HDMI Audio</string>
       </property>
      </widget>
     </item>
     <item row="12" column="7">
      <widget class="QLabel" name="label_43">
       <property name="text">
        <string>Dimmer</string>
       </property>
      </widget>
     </item>
     <item row="13" column="7">
      <widget class="QLabel" name="label_44">
       <property name="text">
        <string>Sleep</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QCheckBox" name="VideoConvCheckBox">
       <property name="text">
        <string>Video Converter On</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3" colspan="2">
      <widget class="QCheckBox" name="VirtualSBCheckBox">
       <property name="text">
        <string>Virtual SB</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3" colspan="2">
      <widget class="QCheckBox" name="VirtualHeightCheckBox">
       <property name="text">
        <string>Virtual Height</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3" colspan="2">
      <widget class="QCheckBox" name="VirtualWideCheckBox">
       <property name="text">
        <string>Virtual Wide</string>
       </property>
      </widget>
     </item>
     <item row="10" column="3" colspan="2">
      <widget class="QCheckBox" name="AnalogInputATTCheckBox">
       <property name="text">
        <string>Analog Input ATT</string>
       </property>
      </widget>
     </item>
     <item row="11" column="3" colspan="2">
      <widget class="QCheckBox" name="MCACCEQCheckBox">
       <property name="text">
        <string>MCACC Equalizer</string>
       </property>
      </widget>
     </item>
     <item row="12" column="3" colspan="2">
      <widget class="QCheckBox" name="StandingWaveCheckBox">
       <property name="text">
        <string>Standing Wave</string>
       </property>
      </widget>
     </item>
     <item row="14" column="3" colspan="2">
      <widget class="QCheckBox" name="DigitalNoiseReductionCheckBox">
       <property name="text">
        <string>Digital Noise Reduction</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6" colspan="2">
      <widget class="QCheckBox" name="HiBitCheckBox">
       <property name="text">
        <string>HiBit</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6" colspan="2">
      <widget class="QCheckBox" name="FixedPCMCheckBox">
       <property name="text">
        <string>Fixed PCM</string>
       </property>
      </widget>
     </item>
     <item row="5" column="6" colspan="2">
      <widget class="QCheckBox" name="SACDGainCheckBox">
       <property name="text">
        <string>SACD Gain +6dB</string>
       </property>
      </widget>
     </item>
     <item row="6" column="6" colspan="2">
      <widget class="QCheckBox" name="AutoDelayCheckBox">
       <property name="text">
        <string>Auto Delay</string>
       </property>
      </widget>
     </item>
     <item row="8" column="6" colspan="2">
      <widget class="QCheckBox" name="LoudnessCheckBox">
       <property name="text">
        <string>Loudness</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3" colspan="2">
      <widget class="QCheckBox" name="VirtualDepthCheckBox">
       <property name="text">
        <string>Virtual Depth</string>
       </property>
      </widget>
     </item>
     <item row="8" column="3" colspan="2">
      <widget class="QCheckBox" name="SoundRetrieverCheckBox">
       <property name="text">
        <string>Sound Retriever</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QComboBox" name="DRCComboBox"/>
     </item>
     <item row="3" column="7">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>DRC</string>
       </property>
      </widget>
     </item>
     <item row="4" column="7">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>LFE ATT</string>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QComboBox" name="LFEATTComboBox"/>
     </item>
     <item row="11" column="6">
      <widget class="QComboBox" name="PQLSComboBox"/>
     </item>
     <item row="11" column="7">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>PQLS</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>11</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
